examples/dataflow-data-generator/README.md

```diff
@@ -136,6 +136,17 @@ Output format is specified by passing one of the `--csv_schema_order` or `--avro
 --avro_schema_file=/path/to/linorders.avsc
 ```
 
+`--write_to_parquet` is a flag that specifies the output should be parquet. In order for beam to write to parquet, 
+a pyarrow schema is needed. Therefore, this tool translates the schema in the --schema_file to
+a pyarrow schema automatically if this flag is included, but pyarrow doesn't support all fields that are supported
+by BigQuery. STRING, NUMERIC, INTEGER, FLOAT, NUMERIC, BOOLEAN, TIMESTAMP, DATE, TIME, and DATETIME types are supported. 
+However BYTE, GEOGRAPHY, and RECORD fields are not supported and cannot be included in the --schema_file when writing
```

Comment:
change request in original PR: https://github.com/GoogleCloudPlatform/professional-services/pull/199#discussion_r262753055

Reply:
BQ parquet loading now supports nested types. However, we can punt on record types until my PR for record types gets merged.